packages/config/tests/parse/snapshots/tests.js.md

```diff
@@ -155,11 +155,10 @@ Generated by [AVA](https://avajs.dev).
 
     `When resolving config file packages/config/tests/parse/fixtures/parse_error/netlify.toml:␊
     Could not parse configuration file␊
-    Unterminated string at row 1, col 3, pos 2:␊
-    1> "␊
-         ^␊
-    2: ␊
+    Invalid TOML document: incomplete key-value: cannot find end of key␊
     ␊
+    1:  "␊
+        ^␊
```

Comment:
👍🏼 Seems fine. I wouldn't consider the exact sub-error message that we print for malformed `netlify.toml` files to be part of the stable public interface of the package.